<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>collectionGUI</class>
 <widget class="QDialog" name="collectionGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1126</width>
    <height>596</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>897</width>
    <height>596</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1500</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>540</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>681</width>
     <height>451</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Item</string>
    </attribute>
    <widget class="QTableWidget" name="tableItem">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>641</width>
       <height>361</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>357</x>
       <y>20</y>
       <width>291</width>
       <height>25</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="line_itemfind"/>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_item">
        <item>
         <property name="text">
          <string>Status</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>OverDue</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Iden Code</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_6">
        <property name="text">
         <string>Find</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Borrower</string>
    </attribute>
    <widget class="QTableWidget" name="tableBorrow">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>641</width>
       <height>361</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>20</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>OverDue</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>162</width>
       <height>25</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Sort : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_6">
        <item>
         <property name="text">
          <string>Name</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Borrowed Item</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="Collection">
    <attribute name="title">
     <string>Collection</string>
    </attribute>
    <widget class="QTableWidget" name="tableColl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>641</width>
       <height>361</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>20</y>
       <width>291</width>
       <height>25</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="lineCollFind"/>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxColl">
        <item>
         <property name="text">
          <string>Type</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Data Type</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Category</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_5">
        <property name="text">
         <string>Find</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>167</width>
       <height>25</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Sort :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_5">
        <item>
         <property name="text">
          <string>Title</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Number Of Item</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_borrow" native="true">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>50</y>
     <width>261</width>
     <height>231</height>
    </rect>
   </property>
   <widget class="QPushButton" name="AddBorrower">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>190</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>220</width>
      <height>186</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_2">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_7">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_10"/>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_8">
       <property name="text">
        <string>Address</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_11"/>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton_9">
       <property name="text">
        <string>Department</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>IT</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BCOMM</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Design</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="pushButton_10">
       <property name="text">
        <string>Mobile</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit_12"/>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="pushButton_11">
       <property name="text">
        <string>Item Borrowed</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEdit_13"/>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="pushButton_12">
       <property name="text">
        <string>Item Late</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="lineEdit_14"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_collection" native="true">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>60</y>
     <width>251</width>
     <height>311</height>
    </rect>
   </property>
   <widget class="QPushButton" name="Add_Collection">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>250</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>296</width>
      <height>257</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_3">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_13">
       <property name="text">
        <string>Type</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBox_2">
       <item>
        <property name="text">
         <string>CD</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>DVD</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_14">
       <property name="text">
        <string>Data Type</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBox_3">
       <item>
        <property name="text">
         <string>Image</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Audio</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Video</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton_15">
       <property name="text">
        <string>Title</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_20"/>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="pushButton_16">
       <property name="text">
        <string>Version</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit_21"/>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="pushButton_18">
       <property name="text">
        <string>Publisher</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEdit_23"/>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="pushButton_19">
       <property name="text">
        <string>Year Recored</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="lineEdit_24"/>
     </item>
     <item row="6" column="0">
      <widget class="QPushButton" name="pushButton_20">
       <property name="text">
        <string>Duration</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="lineEdit_25"/>
     </item>
     <item row="7" column="0">
      <widget class="QPushButton" name="pushButton_21">
       <property name="text">
        <string>Category</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QComboBox" name="comboBox_4">
       <item>
        <property name="text">
         <string>Engineering and Technology</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>General</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Business and Economics</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>History</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Film and media studies</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Psychology</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Images</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>photographs&amp; maps</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="pdf_push">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>540</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>PDF and Print</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
